src/main/java/ch/sbb/polarion/extension/pdf/exporter/weasyprint/service/model/WeasyPrintServiceInfo.java

```diff
@@ -3,7 +3,8 @@
 import lombok.Data;
 
 @Data
-public class WeasyPrintServiceVersion {
+public class WeasyPrintServiceInfo {
     private String python;
     private String weasyprint;
+    private String weasyprintServiceVersion;
```

Comment:
here we need to have `weasyprintService`or rename all other fields to have suffix `Version`